front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/MethodsUsedWidget/typings.ts

```diff
@@ -0,0 +1,6 @@
+import { ChartWidgetProps } from '../typings';
+
+export interface Props extends Omit<ChartWidgetProps, 'projectId'> {
```

Comment:
We are omitting projectId, because this widget is only available for the entire platform?

Reply:
For the moment, yes